### PR TITLE
Fix returning flask Response objects in a tuple

### PR DESCRIPTION
**Pull request by vimalloc**
Allows these flask paradigms to work in connexion endpoints:

```python
return jsonify({'foo': 'bar'}), 200
```

```python
return jsonify({'foo': 'bar'), {'new_header': 'header_value'}
```

```python
return jsonify({'foo': 'bar'}), 201, {'new_header': 'header_value'}
```

### CODE REVIEW

**Comment by rafaelcaricio** [comment]
@vimalloc Could you please write a test for this new code?

**Comment by rafaelcaricio** [comment]
👍

**Comment by hjacobs** [comment]
Stupid question: why would anybody return a tuple where only the first element is actually used (`response[0]`)? Or did I get it wrong?

File: connexion/apis/flask_api.py [hunk 1]
```diff
@@ -208,6 +208,9 @@ def _get_flask_response(cls, response, mimetype):
         if flask_utils.is_flask_response(response):
             return response
 
+        elif isinstance(response, tuple) and flask_utils.is_flask_response(response[0]):
```
Comment (hjacobs) [inline connexion/apis/flask_api.py]: isn't this throwing away all the other tuple elements?

Comment (vimalloc) [inline connexion/apis/flask_api.py]: Here we are only checking if the first element in the tuple is already a flask response, along the lines of (<Response>, status_code, headers). If so, we send the full tuple to the underlying flask code which turns a returned tuple into a flask response: 

https://github.com/pallets/flask/blob/master/flask/app.py#L1698

Comment (hjacobs) [inline connexion/apis/flask_api.py]: Right, my fault

**Pull request merged**